src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,41 @@
+.menu {
+  box-sizing: border-box;
+  height: 100vh;
+  background-color: #e5e6e8;
+  font-weight: 700;
+  text-transform: uppercase;
+
+  &__nav {
+    margin-bottom: 48px;
+  }
+
+  &__top {
+    margin-bottom: 32px;
+  }
+
+  &__phone-number {
+    color: inherit;
+    text-decoration: none;
+    display: block;
+    font-size: 16px;
+    line-height: 140%;
+    letter-spacing: 2px;
+    margin-bottom: 16px;
+  }
+
+  &__call-to-order {
+    width: fit-content;
```

Comment:
Consider using 'display: inline-block;' instead of 'width: fit-content;' for better browser compatibility.